packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -274,13 +267,12 @@ abstract contract Sphinx {
         }
 
         vm.revertTo(snapshotId);
-        deploymentInfo.gasEstimates = _sphinxEstimateMerkleLeafGas(
+        deploymentInfo.gasEstimates = sphinxUtils.estimateMerkleLeafGas(
             parsedAccesses,
-            IGnosisSafe(safe),
-            module
+            address(this)
```

Comment:
I moved the `_estimateMerkleLeafGas` function to `SphinxUtils` to prevent the `Solc.spec.ts` test suite in the demo package from failing due to a "stack too deep" error